### DISCUSSION TITLE
‎

### DISCUSSION THREAD

**Discussion by wwalterr** [bisq-network/bisq#6344] [Q&A] [answered]

**Comment by ghost** [comment, marked as answer]
Have you looked up the transaction ID on a block explorer, such as mempool.space?
You might see that it cannot be found.  That's usually because your SPV wallet is not synced correctly with the blockchain.  
See https://bisq.wiki/Troubleshooting_wallet_issues and https://bisq.wiki/Resyncing_SPV_file
